content/en/blog/news/first-post/index.md

```diff
@@ -1,17 +1,129 @@
 ---
-date: 2024-02-28
+date: 2024-03-13
 title: Introducing SpinKube
 linkTitle: Introducing SpinKube
 description: >
-  A new open source project that streamlines the experience of developing, deploying, and operating Wasm workloads on Kubernetes.
-author: Fermyon Technologies ([@fermyontech](https://twitter.com/fermyontech))
+  Bringing the next wave of hyper-efficient serverless to Kubernetes.
+author: The SpinKube Team ([@SpinKube](https://mastodon.social/@SpinKube))
 resources:
   - src: "**.{png,jpg}"
     title: "Image #:counter"
 ---
+Today we're introducing SpinKube - an open source platform for efficiently 
+running Spin-based WebAssembly applications on Kubernetes. Built with love by 
+folks from Microsoft, SUSE, LiquidReply, and Fermyon.
 
-# Introducing SpinKube
+SpinKube combines the application lifecycle management of the [Spin Operator][spin-operator], the efficiency of the [containerd-shim-spin][containerd-shim-spin], and the node lifecycle management of the forthcoming [runtime-class-manager][runtime-class-manager] (formerly [KWasm][kwasm]) to provide an excellent developer and operator experience alongside excellent density and scaling characteristics regardless of processor architecture or deployment environment.
 
+>
+> TL;DR? Check out the [quickstart][quickstart] to learn how to deploy
+> WebAssembly to Kubernetes today.
+> 
 
+## Why Serverless?
 
+Containers and Kubernetes revolutionized the field of software development and operations. A unified packaging and dependency management system that was flexible enough to allow most applications to become portable - to run the same versions locally as you did in production - was a desperately needed iteration from the Virtual Machines that came before.
 
+But while containers give us a relatively light weight abstraction layer, they come with a variety of issues:
+
+- *Change management* - patching system dependencies (such as OpenSSL) need to happen on every image independently, which can become difficult to manage at large scale. 
+- *Rightsizing workloads* - managing access to shared resources like CPU and memory correctly is difficult, which often leads to over-provisioning resources (over introducing downtime) resulting in low utilization of host resources. 
+- *Size* - containers are often bigger than needed (impacting important metrics like startup time), and often include extraneous system dependencies by default which are not required to run the application
+
+The desire to fix many of these issues is what led to the "first" generation of Serverless.
+
+>
+> Borrowing ideas from CGI, Serverless apps are not written as persistent servers. Instead Server-less code is code that responds to events (such as HTTP requests), but does not run as a daemon process listening on a socket. The networking part of serverless architectures is relegated to the underlying infrastructure.
```

Comment:
Going to reject this one because I want to avoid conflating `component` - but in this case it _is_ lower level (you only shipping your business logic is a good thing 😅 )